src/pg/sql/11_cdb_union_adjacent.sql

```diff
@@ -0,0 +1,43 @@
+CREATE OR REPLACE FUNCTION _cdb_final_union_adjacent( joined_geoms geometry[] )
+RETURNS geometry[] AS $$
+BEGIN
+    RETURN joined_geoms;
+END
+$$ LANGUAGE plpgsql;
+
+
+CREATE OR REPLACE FUNCTION _cdb_state_update_union_adjacent(clusters geometry[], new_geom  geometry)
+RETURNS geometry[] AS $$
+DECLARE
+  joins  geometry[] :='{}';
```

Comment:
should we limit the signature to `MULTIPOLYGON`?  I think this would break with points...